python/runtime/cudaq/platform/py_alt_launch_kernel.cpp

```diff
@@ -233,24 +242,49 @@ pyAltLaunchKernelBase(const std::string &name, MlirModule module,
 
   // If we have any state vector data, we need to extract the function pointer
   // to set that data, and then set it.
-  for (auto &[stateHash, svdata] : *stateStorage) {
-    if (svdata.kernelName != name)
+  for (auto &[stateHash, stateData] : *stateStorage) {
+    if (stateData.kernelName != name)
       continue;
+
     auto setStateFPtr = jit->lookup("nvqpp.set.state." + stateHash);
-    if (!setStateFPtr)
-      throw std::runtime_error(
-          "python alt_launch_kernel failed to get set state function.");
+    if (auto error = setStateFPtr.takeError()) {
```

Comment:
Note: using `!setStateFPtr` just fails with a message about forgetting to check the result of `Expected<E>`. Updated to take the error and report the actual error message (here and the new code below)